pkgs/development/python-modules/dagger-io/default.nix

```
@@ -0,0 +1,61 @@
+{ lib
+, anyio
+, attrs
+, beartype
+, buildPythonPackage
+, cattrs
+, colorama
+, fetchFromGitHub
+, gql
+, graphql-core
+, httpx
+, platformdirs
+, poetry-core
+, pythonOlder
+, rich
+, shellingham
+, strawberry-graphql
+, typer
+, typing-extensions
+}:
+
+buildPythonPackage rec {
+  pname = "dagger-io";
+  version = "v0.6.1";
```

```suggestion
  version = "0.6.1";
```
`v` is a prefix, and it should not be included in the version.